indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -20,36 +21,65 @@ import {
 } from 'indico/react/components';
 import {useNumericParam} from 'indico/react/util/routing';
 import {Translate} from 'indico/react/i18n';
+import {useIndicoAxios} from 'indico/react/hooks';
 import {EditableType} from '../../models';
 import StateIndicator from '../../editing/timeline/StateIndicator';
 
 import './EditableList.module.scss';
 
 export default function EditableList() {
-  const list = []; // Replace it with list from server
   const eventId = useNumericParam('confId');
   const {type} = useParams();
-  const [sortBy, setSortBy] = useState('id');
+  const {data, loading: isLoadingEditableList, lastData} = useIndicoAxios({
+    url: editableListURL({confId: eventId, type}),
+    camelize: true,
+    trigger: eventId,
+  });
+
+  const editableList = data || lastData;
+  if (isLoadingEditableList && !lastData) {
+    return <Loader inline="centered" active />;
+  } else if (!editableList) {
+    return null;
+  }
+  editableList.forEach(contribution => {
+    if (_.isEmpty(contribution.editable)) {
```

Comment:
`contribution.editable === null` should be enough here

Reply:
I have removed this, can you check again?